plugins/transformations/sequence2array/srcCxx/Sequence2ArrayTransformation.cxx

```diff
@@ -194,7 +172,7 @@ bool Seq2ArrayTransformation::are_type_compatible(
         }
```

Comment:
Why does it matter if the output uses different "dimensions"? At the end of the day, all that matters in order to be able to convert an array member is that the output array is bigger or equal in "total size".

I would get rid of this check altogether.

Reply:
This transformation does nothing with arrays, it only performs the transformation of sequences into 1D arrays. Then this checking is to ensure that the input and output array are the same (implying the same size).